src/mmw/js/src/controllers.js

```diff
@@ -26,6 +27,34 @@ var AppController = {
     }
 };
 
+var ErrorHandlers = {
+    itsi: function() {
+        router.navigate('');
+        alert("We're sorry, but there was an error logging you in with your" +
```

Comment:
As part of this PR, can you create an issue for @ctaylo37 to give us some markup for a general purpose error modal?